README.md

```diff
@@ -15,7 +15,8 @@ The modules are:
 These modules may be broken out to individual repositories at some time, but
 they are colocated for convenience and shared evolution for now.
 
-# api
+# Docker Compose Test Suites
+## docker-compose-api.yml
```

Comment:
I'm not sure what splitting the files gets us. Could we do it with profiles on the services in one file? It's a command line argument either way, but a single file might be more clear for inspection and maintenance.

Reply:
Umm, well, yes.  I not too sure how that would be done, so pairing on that would help me out. The other thing was making sure the DB was available because the API executes a rake db:migrate.

Reply:
OK, cool. Setting up profiles and depends_on is easy enough. The trick is waiting a little bit for the DB to be ready before the migrate. There are a few ways to do that, the simplest being something like `sleep 5`. MariaDB starts up quite fast, even on the first launch. The more robust approach uses something like [dockerize](https://github.com/jwilder/dockerize).